tests/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ function(add_uncrustify_test name lang config rerun_config input result output)
   )
 endfunction()
 
-foreach(test_lang c-sharp c cpp d java pawn objective-c vala ecma)
+foreach(test_lang c-sharp c cpp d java pawn objective-c vala ecma imported)
```

Comment:
I haven't noticed it in the earlier already closed PRs: Why are we making an extra category for this tests?